Games/types/Mafia/roles/cards/OrangeGiver.js

```diff
@@ -1,15 +1,23 @@
 const Card = require("../../Card");
 const { PRIORITY_ITEM_GIVER_DEFAULT } = require("../../const/Priority");
+const { EXCLUSIVE_PRIORITY_HOT_SPRINGS } = require("../../const/MeetingExclusivePriority");
 
 module.exports = class OrangeGiver extends Card {
 
     constructor(role) {
         super(role);
 
         this.meetings = {
+            "Hot Springs": {
+                states: ["Night"],
+                flags: ["exclusive", "group", "speech", "anonymous"],
+                priority: EXCLUSIVE_PRIORITY_HOT_SPRINGS,
+            },
+
             "Give Orange": {
                 states: ["Night"],
                 flags: ["voting"],
+                priority: EXCLUSIVE_PRIORITY_HOT_SPRINGS + 1,
```

Comment:
this allows capy to still give oranges despite the hotspring meeting being exclusive